src/admin/paths/admin@expense_policies.yaml

```diff
@@ -0,0 +1,98 @@
+post:
+  tags:
+    - Expense Policies
+  summary: Create or Update Expense Policies
+  description: |
+    Create or Update Expense Policies <br>
+    **Apart from the required fields mentioned below, the request body should contain the following:**
+    - At least one condition field. You can choose any condition from expense_* or employee_* or both.
+    - At least one action field. You can choose any action from action_*.
```

Comment:
You can choose any action from action_*.

You should choose at least one condition from action_*.

Reply:
can remove the first sentence altogether